src/systems/systems/src/simulation/mod.rs

```diff
@@ -457,7 +489,40 @@ pub trait Write<T> {
     ///     }
     /// }
     /// ```
-    fn write(&mut self, name: &str, value: T);
+    fn write(&mut self, name: &str, value: T)
+    where
+        Self: Sized + Writer,
+    {
+        let value = self.convert(value);
+        self.write_f64(name, value)
+    }
+
+    /// Write an ARINC 429 value to the simulator.
+    ///
+    /// Note that the `f64` will be converted to a `f32` internally, thus reducing precision.
+    /// # Examples
+    /// ```rust
+    /// # use systems::simulation::{SimulationElement, SimulationElementVisitor,
+    /// #    SimulatorReader, SimulatorWriter, Write};
+    /// # use systems::shared::arinc429::SignStatus;
+    /// struct MySimulationElement {
+    ///     n: f64,
+    /// }
+    /// impl SimulationElement for MySimulationElement {
+    ///     fn write(&self, writer: &mut SimulatorWriter) {
+    ///        writer.write_arinc429("MY_SIMULATOR_ELEMENT_N", self.n, SignStatus::NormalOperation);
+    ///     }
+    /// }
+    /// ```
+    fn write_arinc429(&mut self, name: &str, value: T, ssm: SignStatus)
```

Comment:
I find it a bit weird that `read_arinc429` returns a `Arinc429Word`, however `write_arinc429` requires the raw parameters making up an `Arinc429Word`. As we're probably going to introduce additional parameters here, like SDI and the Label, how about we just write an `Arinc429Word` itself? Then, if we make `Arinc429Word`  more complicated, we don't have go back and refactor all the `write_arinc429` calls.

Reply:
In the PR description I wrote:

> The SimVar's name can be considered equivalent to the ARINC 429 label and SDI, in that it can without restrictions indicate what value the SimVar contains, and additionally can indicate what is the source of the information (e.g. `"ADIRS_1_...", "ADIRS_2_...").

Do you feel this statement is not entirely correct for our situation?

I'd like to note that likely soon your Arinc429 bus changes will change the way we write these SimVars (component -> bus -> simvar). For that situation I think having a `write_arinc429` function which accepts an `Arinc429Word` makes sense, as you're dealing with that type within the bus itself. However, at this very moment having a `write_arinc429` function which is somewhat equivalent to the current `write` feels _okay_ to me. This area probably is where our implementations are "colliding".

Reply:
You're right, this is redundant.